src/consts/interfaces.consts.ts

```diff
@@ -76,6 +76,8 @@ export interface ICreateLinkParams {
 	linkDetails: IPeanutLinkDetails
 	peanutContractVersion?: string
 	password?: string
+	recipient?: string // only this address will be able to claim if set
```

Comment:
Yeah, I agree. Idk how we could change the variable name (and I would not change it since it's already called `reclaimableAfter` in the smart contract), but at least I changed the comment